src/configurations/cancercomplexity/routesConfig.ts

```diff
@@ -275,6 +356,18 @@ const routes: GenericRoute[] = [
                   sql: publicationSql,
                   sqlOperator: 'LIKE',
                   synapseConfigArray: [
+                    {
+                      name: 'CardContainerLogic',
+                      columnName: 'publicationId',
+                      title: 'Related People',
+                      tableSqlKeys: ['publicationId'],
+                      props: {
+                        sqlOperator: 'LIKE',
```

Comment:
Does sqlOperator '=' not work here?

Reply:
Oh yes, you are right - using `=` works here.